gum/backend-darwin/gumkernel-darwin.c

```diff
@@ -688,6 +731,20 @@ gum_kernel_get_version (void)
 
 #ifdef HAVE_ARM64
 
+static bool
```

Comment:
For consistency we should avoid C99 types:

```suggestion
static gboolean
```